addons/point_of_sale/static/src/js/models.js

```diff
@@ -145,6 +145,7 @@ export class PosGlobalState extends PosModel {
             selectedOrder: null,
             selectedPartner: null,
             selectedCategoryId: null,
+            searchProductWord: "",
```

Comment:
Can you add the following comment? `// FIXME POSREF this piece of state should probably be private to the product screen but it currently needs to be available to the ProductInfo screen for dubious functional reasons`